Assets/Prefabs/Game/Arrow.prefab

```diff
@@ -80,17 +83,21 @@ MonoBehaviour:
   SyncPositionX: 1
   SyncPositionY: 0
   SyncPositionZ: 1
-  SyncRotAngleX: 0
+  SyncRotAngleX: 1
```

Comment:
do we need those other axis? Does it change anything actually, with the quaternion optims available?

Reply:
When quaternion synchronization is set, all axis are synchronized and the NetworkTransformEditor inspector view disables the selection of those axis (i.e. it shouldn't matter what those values are). 
_With the quaternion compression enabled, it reduces the entire per delta update from 12 bytes (i.e. all Euler axis) down to 4 bytes. So, the savings should be from around 21 bytes to roughly ~12 bytes per arrow update depending upon how good of compression the tick and bitset flags are._

Reply:
ok ok I see. I was just reviewing from github diff, so didn't see the UI change :P 
I'm wondering, if we were to use per axis rot syncing and half floats, wouldn't that be reduced to 2 bytes instead of 4? Since we're only rotating around y?

Reply:
If the arrow only uses the y axis for rotation then yes... you would want to do that instead. _(and the savings would be the same as the Character.prefab)_